py/desispec/scripts/fluxcalibration.py

```diff
@@ -77,9 +77,14 @@ def main(args) :
 
     # check that the model_fibers are actually standard stars
     fibermap = frame.fibermap
-    model_fibers = model_fibers%500
-    if np.any(fibermap['OBJTYPE'][model_fibers] != 'STD'):
-        for i in model_fibers:
+
+    ## get the fiber indices
+    w = np.array([objtype.strip() == 'STD' for objtype in fibermap['OBJTYPE'][:]])
+    fibers = fibermap['FIBER'][:][w]
+    if np.any(fibers != model_fibers):
+        i_model_fibers = list(fibers)
+        i_model_fibers = [model_fibers.index(i) for i in model_fibers]
```

Comment:
This line is causing using test errors with the message
```
Traceback (most recent call last):
  File "/Users/sbailey/desi/git/desispec/bin/desi_compute_fluxcalibration", line 16, in <module>
    fluxcal.main(args)
  File "/Users/sbailey/desi/git/desispec/py/desispec/scripts/fluxcalibration.py", line 86, in main
    i_model_fibers = [model_fibers.index(i) for i in model_fibers]
  File "/Users/sbailey/desi/git/desispec/py/desispec/scripts/fluxcalibration.py", line 86, in <listcomp>
    i_model_fibers = [model_fibers.index(i) for i in model_fibers]
AttributeError: 'numpy.ndarray' object has no attribute 'index'
```
This happens within test_binscripts.py resulting in the final test error message
```
======================================================================
FAIL: test_compute_fluxcalib (__main__.TestBinScripts)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "py/desispec/test/test_binscripts.py", line 203, in test_compute_fluxcalib
    self.assertEqual(err, 0)
AssertionError: 1 != 0
```
Please check this code.  Note that line 86 blows away the `i_model_fibers` defined in line 85.  You might need to use `np.where(model_fibers == i)[0]` instead of `model_fibers.index(i)`.

Please also run `python setup.py test` to confirm that the unit tests run locally.